cypress.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
que es?

Reply:
oh I think this might be one of the many files cypress loads

Reply:
I think you are correct about the cypress files. We can reach out and see what we actually need to keep.